go.mod

```diff
@@ -1,4 +1,4 @@
-module gremgo
+module github.com/schwartzmx/gremgo
```

Comment:
@elfido should we just be able to change the module path in `go.mod` to the full url,  as described here, https://github.com/golang/go/wiki/Modules#gomod
`module github.com/schwartzmx/gremgo-neptune`?

Reply:
can we try that and I can make another release?

Reply:
Will give a try, thanks. I'll keep you posted